crates/runner-host/crates/web-client/data/index.scss

```diff
@@ -23,10 +23,49 @@ svg {
 }
 
 .button {
```

Comment:
Is it possible to add some margin (or any other CSS technique) to get the buttons and LEDs horizontally centered in those 79 pixels of the left panel?

Reply:
Definitely, should be centered now.